cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -165,6 +173,46 @@ def _resolve_target(self, target: Union[str, None]) -> str:
 
         return target
 
+    def _validate_cirq_circuit(self, circuits: Union[cirq.Circuit, Sequence[cirq.Circuit]]) -> None:
+        """Validates that the input is either a single cirq.Circuit or a list of cirq.Circuit
+        instances.
+
+        Args:
+            circuit: The circuit to run.
+
+        Raises:
+            InvalidInputError: If the 'circuits' input is not a cirq.Circuit or a list of
+            cirq.Circuit instances.
+            InvalidInputError: If the 'circuits' input is a list containing elements that
+            are not cirq.Circuit instances.
```

Comment:
nits: (1) i think the second case is covered by the first, (2) code should be wrapped in backticks, and (2) subsequent lines should be indented once more
```suggestion
            InvalidInputError: If the input is not a `cirq.Circuit` or a list of `cirq.Circuit`
                instances.
```